src/routes/gifs.js

```diff
@@ -104,17 +105,37 @@ const createComment = catchAsync( async (req, res) => {
 
 })
 
+const recordLikesOnGif = catchAsync( async(req, res) => {
+	const {id} = req.params
+	const { userId } = req.body
+	const newLike = await postService.recordPostLikes({
+		userId,
+		id
+		 })
+	res.status(201).json({
+		status: 'success',
+		data: {
+			message: 'GIF image successfully liked',
+			...transformGifResponse(newLike)
```

Comment:
Kindly transform this on its own since an inserted `like` is not really the same as a inserted Gif,